indigo_app/static/javascript/indigo/models.js

```diff
@@ -185,6 +187,8 @@
       // changes in a single revision on the server.
       // We do this by delegating to the document object.
       this.document.attributes.content = this.get('content');
```

Comment:
when in provision mode, wrap this in appropriate tags to make it a fully valid akn portion